Transactions clean up (mostly list view)

morebrownies opened the pull request:
**Transactions**
- Cleaned up the layout of the sales, purchases and cases tab
- Added buttons to the right side of the list view
- Added count total to sales tab 
- Added count total to purchases tab 
- Sort sales by most recent by default
- Sort purchases by most recent by default
- Cleaned up some copy
- Added icons to the tabs in the modal
- If digital item, display "Confirm order" or if physical item display "Mark as shipped" in button

**User page**
- Hide less important buttons on user page and added a control to view more
- If user does not have handle, show the guid under their name

**Bugs**
- Fixes #609
- Fixes #585

ilxwolf committed:
address bar larger text:

ilxwolf committed:
Merge branch 'master' into transactions

ilxwolf committed:
changes

ilxwolf committed:
Merge branch 'master' into transactions

ilxwolf committed:
Merge branch 'master' into transactions

ilxwolf committed:
button slide out + transactions clean up

ilxwolf committed:
changes

ilxwolf committed:
changes

ilxwolf committed:
changes

ilxwolf committed:
fixes #585

jjeffryes commented on js/templates/transactionModal.html:
```diff
@@ -43,7 +43,11 @@ <h3 class="txt-bright"><%= polyglot.t('Loading') %> </h3>
       <a class="btn btn-txt custCol-secondary pull-right js-showConfirmForm">
         <span class="ion-checkmark fontSize11 marginRight2 textOpacity1"></span>
-        <%= polyglot.t('transactions.ConfirmOrder') %>
+        <% if(ob.vendor_offer.listing.metadata.category == "digital good") { %>
```
This should be digital good or service.

morebrownies replied:
Fixed

ilxwolf committed:
fixed issue

jjeffryes committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into transactions

jjeffryes committed:
Show Cases when user becomes moderator

- when user becomes a moderator, the cases link appears in the nav menu
- when user stops being a moderator, the cases link is removed from the
nav menu
- added briefcase icon to cases link and cases tab
